tests/test_random_number_generator.h

```diff
@@ -106,6 +105,81 @@ TEST_CASE("[RandomNumberGenerator] Restore from seed") {
 	CHECK_MESSAGE(s0_1_before == s0_1_after, msg);
 	CHECK_MESSAGE(s0_2_before == s0_2_after, msg);
 }
+
+TEST_CASE("[RandomNumberGenerator] Check rand range for expected functionality") {
+	Ref<RandomNumberGenerator> rng = memnew(RandomNumberGenerator);
+	rng->set_seed(0);
+	uint64_t prev_state = rng->get_state();
+	uint32_t seed_before = rng->get_seed();
+
+	String msg = "Randf range should give float between -100 and 100, base test";
+	float num_before = rng->randf_range(-100, 100);
+	CHECK_MESSAGE(-100 <= num_before, msg);
```

Comment:
I know that this makes sense when using mathematical notation (`-100 <= n <= 100`), but since this is assertions which are spread across different lines, I think it's better to have the variable be the first operand (just like for `if (n >= 100)` conditional checks, for instance).

```suggestion
	CHECK_MESSAGE(num_before >= -100, msg);
```

The same applies to other assertions.